youtube_dl/extractor/pornhub.py

```diff
@@ -165,6 +165,14 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         host = mobj.group('host') or 'pornhub.com'
         video_id = mobj.group('id')
+        cookie_file = self._downloader.params.get('cookiefile')
+
+        if host == 'pornhubpremium.com':
```

Comment:
Should also match `pornhubpremium.net`. Or extractor should not match `pornhubpremium.net`.

Reply:
Updated.

Where is `pornhub.net` used in the wild? For me it just redirects to `.com`.

Reply:
Not quite sure. Currently it redirects to `pornhub.com`. Possibly this was not the case the past.